src/connectors/powered-by/connectPoweredBy.ts

```diff
@@ -47,7 +47,7 @@ const connectPoweredBy: PoweredByConnector = function connectPoweredBy(
     'utm_campaign=poweredby';
 
   return widgetParams => {
-    const { url = defaultUrl } = widgetParams || {};
+    const { url = defaultUrl } = widgetParams || ({} as typeof widgetParams);
```

Comment:
how come this isn't PoweredByConnectorParams or something? makes sense either way though, since all keys are optional

Reply:
It's actually supposed to be `PoweredByWidgetParams` type, but 

```
export type PoweredByConnector = Connector<
  PoweredByRendererOptions,
  PoweredByConnectorParams
>;
```

We don't explicitly have it at the connector definition.